_component_design/choice-tiles.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Choice Tiles
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+preview-image: preview-images/choice-tile.svg
+status: stable
+resource: true
+last-modified: 2017-11-12
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+### When to use
+
+- Use choices tiles when presenting a set of choices that are easily recognized by iconography.
+- Only use choice tiles when two or more selections are available.
+
+### Best practices
+
+- When using a description keep the text brief to ensure users engage with are not overwhelmed by a wall across multiple choice tiles.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The choice tile pattern"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/inputs-and-controls/choice-tiles/choice-tile-hero.png)
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Default choice tile
```

Comment:
Change this to `Composition` and make it an H3 inside the left column.

Reply:
updated.